hack/update-estimator-protobuf.sh

```diff
@@ -61,6 +61,15 @@ ${GOPATH}/bin/go-to-protobuf \
   --proto-import="${KARMADA_ROOT}/vendor" \
   --proto-import="${KARMADA_ROOT}/third_party/protobuf"
 
+# go-to-protobuf has a argument:
+#    -o, --output-base string        Output base; defaults to $GOPATH/src/
+# So the protobuf code generated to ${GOPATH}/src/github.com/karmada-io/karmada by default
+# But we usually fork the karmada repo from karmada-io/karmada, we may not put the root path to ${GOPATH}/src/github.com/karmada-io/karmada in local file system
+# In this case, we need move the generated file from ${GOPATH}/src/github.com/karmada-io/karmada to ${KARMADA_ROOT}
+if [[ "${KARMADA_ROOT}" != "${GOPATH}/src/github.com/karmada-io/karmada" ]]; then
+  mv -f "${GOPATH}"/src/github.com/karmada-io/karmada/pkg/estimator/pb/* "${KARMADA_ROOT}"/pkg/estimator/pb/
+fi
```

Comment:
Can we detect the generated location instead of moving the file?

By the way, it is concerning if just moving files from `${GOPATH}"/src/github.com/karmada-io/karmada/pkg/estimator/pb/*`, because the files might be outdated, not the one just generated.

Reply:
got it